docs/dsom_ansible_review.md

```
@@ -0,0 +1,87 @@
+* On **limited** or unprivileged nodes (e.g., in sandboxed or containerized user environments), ASIMP skips destructive low-level remediations (such as writing sysctls or manipulating restricted hardware services like `auditd`) and shifts execution strictly to audit-only or fallback mode.
+* ASIMP's safety check suite ensures that whenever `asimp_privilege_level == 'full'`, a thorough **Pre-Remediation Safety Check** is run (verifying disk space, SSH syntax, fstab mounts) to guarantee that host-level control actions do not trigger service lockouts.
+
+---
+
+## 🔐 Sovereign Secrets Injection Protocol
+
+DSOM utilizes a **Runtime Injection Pattern** to avoid committing credentials:
+1. `inventory/hosts.yml` contains public configuration variables and placeholder structures.
+2. Production credentials live in git-ignored `vault/production_secrets.yml`.
+3. Commands load secrets dynamically at runtime:
+   ```bash
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)